msbuild/Xamarin.MacDev.Tasks/Tasks/LinkNativeCodeTaskBase.cs

```diff
@@ -75,6 +75,9 @@ bool ExecuteUnsafe ()
 
 			var arguments = new List<string> ();
 			arguments.Add ("clang++");
+			// check if needs to be removed: https://github.com/xamarin/xamarin-macios/issues/18556
+			arguments.Add ("-Xlinker");
+			arguments.Add ("-ld64");
```

Comment:
In beta 5 this was renamed to `ld_classic`. The new linker is still as broken as ever.

Reply:
Good lord apple